client/service/src/client/client.rs

```diff
@@ -865,9 +861,15 @@ impl<B, E, Block, RA> Client<B, E, Block, RA> where
 			// block.
 			(true, ref mut storage_changes @ None, Some(ref body)) => {
 				let runtime_api = self.runtime_api();
+				let execution_context = if import_block.origin == BlockOrigin::NetworkInitialSync {
```

Comment:
I think we should follow the comment in the source for the context:
```
	/// Context for general importing (including own blocks).
	Importing,
	/// Context used when syncing the blockchain.
	Syncing,
```

Reply:
I assumed that `Importing` is for any block import (not only for own blocks). This is the same distinction we make in the informant when we print `Syncing` or `Imported`, the only difference is the queue size.

Reply:
But we always sync :D Even after the initial sync. The point is that this is a security consideration (not trusting foreign blocks).